wolfssl/wolfcrypt/asn_public.h

```diff
@@ -336,6 +338,9 @@ WOLFSSL_API int wc_SetSubject(Cert*, const char*);
 #ifdef WOLFSSL_ALT_NAMES
     WOLFSSL_API int wc_SetAltNames(Cert*, const char*);
 #endif
+
+WOLFSSL_API void wc_DecodedCert_Cache_Free(Cert* cert);
```

Comment:
Might it be cleaner to just have this called `wc_FreeCert` to match the `wc_InitCert`? Or since it only is used after calling the `wc_Set*` API's maybe `wc_SetCert_Free()`? 

Also it looks like `wc_DecodedCert_Cache_Free` should only be a public API if `WOLFSSL_DECERT_CACHE` is not defined.

Reply:
Agreed